i18n/en.json

```diff
@@ -1825,11 +1825,11 @@
   },
   {
     "id": "api.team.import_team.unavailable.app_error",
-    "translation": "Filesize unavilable"
+    "translation": "Filesize unavailable"
```

Comment:
Looks like that error occurs when the importTeam API receives a malformed for submission that is missing the filesize field. I suppose you could say something like:

```Malformed request: filesize field is not present.```